test/e2e/docs/writing_tests.md

```diff
@@ -9,149 +9,130 @@ Refer to the [style guide](docs/style-guide.md) for coding style information.
 <!-- TOC -->
 
 - [Writing Tests](#writing-tests)
-  - [Table of contents](#table-of-contents)
-  - [Selector](#selector)
-  - [Component](#component)
-  - [Page](#page)
-    - [Structure](#structure)
-    - [Guidelines](#guidelines)
-  - [Flow](#flow)
-    - [Structure](#structure)
-    - [Guidelines](#guidelines)
-  - [Gutenberg Blocks](#gutenberg-blocks)
+    - [Table of contents](#table-of-contents)
+    - [Get Started](#get-started)
+    - [Top-level block](#top-level-block)
+    - [Child-level block](#child-level-block)
+    - [Setup](#setup)
+    - [Test step](#test-step)
+    - [Variable naming](#variable-naming)
 
 <!-- /TOC -->
 
-## Selector
+## Get Started
 
-Selectors form the core of any automated e2e test scripts. For a quick overview of selectors, please refer to the [MDN page on CSS selectors](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors).
+Create a spec file under `test/e2e/specs` in the appropriate directory. 
+Follow this general structure when naming a file:
 
-For modern automated e2e tests, `CSS` selectors are mainstream, although `xpath` and text selectors are also sometimes used.
+`wp-<feature>__<subfeature or suite>-spec.js`
```

Comment:
Can we make this show it can be optionally TypeScript too? To encourage folks to use TS?